lib/node_modules/@stdlib/blas/ext/base/gfill-by/benchmark/benchmark.js

```diff
@@ -43,15 +43,12 @@ function createBenchmark( len ) {
 	return benchmark;
 
 	function benchmark( b ) {
-		var f;
 		var y;
 		var i;
 
-		f = uniform( -10.0, 10.0 );
-
 		b.tic();
 		for ( i = 0; i < b.iterations; i++ ) {
-			y = gfillBy( x.length, x, 1, f );
+			y = gfillBy( x.length, x, 1, uniform( -100, 100 ) );
```

Comment:
This change is not desired, as now you are also measuring how quickly `uniform` can return a function.